docs/installation.rst

```diff
@@ -26,6 +26,20 @@ package index:
 
     pip install sectionproperties
 
+
+Installing ``PARDISO`` Solver
+-----------------------------
+
+The default sparse solver used in ``scipy`` is ``SuperLU``.
+It performs okay for small matrices but appears to be very slow for larger matrices.
+The ``PARDISO`` solver is a much faster alternative, but it requires the installation of the ``MKL`` library, which takes a lot of disk space.
```

Comment:
This section reads really well, thanks! I would just suggest providing a link the the pypardiso GitHub page so users can easily decide whether or not they would like to install this. For example:

> The ``PARDISO`` solver is a much faster alternative (see [PyPardiso](https://github.com/haasad/PyPardisoProject)), but it requires...